config/webpack.config.js

```diff
@@ -344,6 +352,15 @@ module.exports = function (env, ilibAdditionalResourcesPath) {
 							}),
 							sideEffects: true
 						},
+						// Adds support for CSS Modules, but using SASS
+						// using the extension .module.scss or .module.sass
+						{
+							test: /\.module\.(scss|sass)$/,
```

Comment:
Are we supporting only modular Sass/SCSS?

Reply:
I will adding test: /\.(scss|sass)$/, soon